### PR TITLE
Cclave 2207/airlock new UI

### PR DESCRIPTION
**Pull request by FabianDEV08**
## Overview

The following PR contains changes to update the UI if the web wormhole app.

## Current Behavior

The app has the UI of the original web wormhole app.

## Expected Behavior

The app should update the UI to accordingly to the figma designs.

## Changes

- Added required css styles.
- Updated index.html according with the new UI.
- Updated main.js, removed unnecesary variables that were linked to deleted html elements, not needed in the new UI.

## Resources 

Screen Shot 2021-11-19 at 1 43 05 PM
Screen Shot 2021-11-19 at 1 43 31 PM

### CODE REVIEW

**Comment by franc5** [comment]
Please, do not include GraphikRegular.otf

**Comment by franc5** [comment]
Please, include the license for ~~GraphikRegular.otf~~ OverpassMono-Bold.ttf (**EDITED**)

File: web/style.css [hunk 1]
```diff
@@ -23,6 +28,16 @@
 	--palette-7: #e796ea; /* purple */
 }
 
+@font-face {
+font-family: "Graphik";
+src: url(assets/fonts/GraphikRegular.otf);
+}
```
Comment (franc5) [inline web/style.css]: Please, remove these lines. See also https://github.com/SpiderOak/webwormhole/pull/7#issuecomment-975576561

**Comment by franc5** [comment]
If the user enters an invalid phrase, there's no way to go back an enter a new one. Let's add a "Enter another phrase" button to allow our users to do so.

Also, please note that this should be "Not a valid Airlock phrase."

Screenshot 2021-11-22 at 11 27 40

**Comment by franc5** [comment]
Once the file is successfully downloaded, we show a "Disconnected." message. Let's change it to something like "File successfully downloaded". Also, let's add an "Enter another phrase" button (the same button suggested in https://github.com/SpiderOak/webwormhole/pull/7#issuecomment-975586107) to allow our users to enter a new phrase.

**Comment by franc5** [comment]
If the user enters an expired ~~wormhole~~ airlock phrase, the following message is shown "No such slot. The wormhole might have expired.". I guess this should be "No such slot. The phrase might have expired.", but let's ask Zach about the correct wording in these case.

Similar to added comments, please add an "Enter another phrase" button.

**Comment by franc5** [comment]
During the "Connecting..." phase, we don't show the filename nor the file size in the file card. Don't we have these values? If so, let's write something like "Loading file..." instead of its name. Please, validate this with Zach before continuing.

Screenshot 2021-11-22 at 11 36 21

**Comment by FabianDEV08** [comment]
> No such slot. The phrase might have expired.

when connecting the file download is not triggered instantly, when the data for the file is obtanied then the name and the size appear, I can add the "Loading file..." text to the file card to make more sense.

**Comment by brownbike** [comment]
Small style thing with the Get File button. You have some default styles causing a split border color.
Screen Shot 2021-11-22 at 10 08 54 AM
Looking at the figma, we should have a border.

**Review by brownbike — APPROVED** [review]
Looks good, I tested this on MacOS

File: web/style.css (hunk 1 above)
Comment (FabianDEV08) [reply web/style.css]: The font has been removed.

**Comment by FabianDEV08** [comment]
> If the user enters an expired ~wormhole~ airlock phrase, the following message is shown "No such slot. The wormhole might have expired.". I guess this should be "No such slot. The phrase might have expired.", but let's ask Zach about the correct wording in these case.
> 
> Similar to added comments, please add an "Enter another phrase" button.



> Once the file is successfully downloaded, we show a "Disconnected." message. Let's change it to something like "File successfully downloaded". Also, let's add an "Enter another phrase" button (the same button suggested in [#7 (comment)](https://github.com/SpiderOak/webwormhole/pull/7#issuecomment-975586107)) to allow our users to enter a new phrase.

**Comment by FabianDEV08** [comment]
Closed it by mistake, this PR is still active.

**Comment by FabianDEV08** [comment]
> Please, do not include GraphikRegular.otf

The font has been removed.

**Comment by FabianDEV08** [comment]
> Please, include the license for ~GraphikRegular.otf~ OverpassMono-Bold.ttf (**EDITED**)

I've just added the files for the license, please check it out and let me know if it's ok, it's my first time adding a font license and I copied what we have in the share-ui repo.

**Comment by FabianDEV08** [comment]
> Small style thing with the Get File button. You have some default styles causing a split border color. Screen Shot 2021-11-22 at 10 08 54 AM Looking at the figma, we should have a border.

I've updated the style to get rid of the weird border.

**Comment by franc5** [comment]
The site looks good. Approving this PR

**Review by franc5 — APPROVED** [review]
The

**Pull request merged**